javascript/packages/orchestrator/src/configGenerator.ts

```diff
@@ -239,7 +239,7 @@ export async function generateNetworkSpec(
       const isCumulusBased =
         parachain.cumulus_based !== undefined
           ? parachain.cumulus_based
-          : ![DEFAULT_ADDER_COLLATOR_BIN, UNDYING_COLLATOR_BIN].includes(
+          : ![DEFAULT_CUMULUS_COLLATOR_BIN, UNDYING_COLLATOR_BIN].includes(
```

Comment:
```suggestion
          : ![DEFAULT_ADDER_COLLATOR_BIN, UNDYING_COLLATOR_BIN].includes(
```
This should test agains adder and undying collators.